test/e2e/project.mk

```diff
@@ -0,0 +1,34 @@
+# Project specific values
+OPERATOR_NAME?=configuration-anomaly-detection
+
+HARNESS_IMAGE_REGISTRY?=quay.io
```

Comment:
if you want, you can sed replace `HARNESS` with `E2E_SUITE` and `harness` to `e2e-suite` - we are trying to move away from the harness term on this side (and just call it a test suite). The naming doesn't matter that much though, just don't want to cause confusion.